src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,45 +1,45 @@
 package edu.kis.vh.nursery;
 
-import edu.kis.vh.nursery.list.IntLinkedList;
+import edu.kis.vh.nursery.stack.IntLinkedList;
 
 
 public class DefaultCountingOutRhymer {
 
-	//private IntArrayStack stack;
-	private IntLinkedList list;
+	//private IntArrayStack stackImplementation;
+	private IStack stackImplementation;
```

Comment:
suffix  "Implementation" of the attribute name is wrong and does not show the meaning.